sql/updates/latest-dev.sql

```diff
@@ -0,0 +1,89 @@
+CREATE OR REPLACE FUNCTION _timescaledb_functions.set_policy_scheduled(hypertable REGCLASS, policy_type TEXT, scheduled BOOL)
```

Comment:
These are included first in the generated files, so I think you need to do the following for all functions:

```suggestion
CREATE FUNCTION _timescaledb_functions.set_policy_scheduled(hypertable REGCLASS, policy_type TEXT, scheduled BOOL)
```

Reply:
👍  Thanks Mats, this was the clue that I needed! It looks like the tests will pass now.